app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
```

Comment:
All below code be simplified to one if statement. Remove unnecessary return and raises. Instead check if command is correct i one if and then nest the code under this condition if it is satisfied, if it's not satisfied - don't do anything.